src/CC-BY-4.0.xml

```diff
@@ -26,7 +26,7 @@
          license they choose before applying it. Licensors should also secure all rights necessary before
          applying our licenses so that the public can reuse the material as expected. Licensors should clearly
          mark any material not subject to the license. This includes other CC-licensed material, or material
-         used under an exception or limitation to copyright. More considerations for licensors.</p>
+         used under an exception or limitation to copyright. More considerations for licensors<alt match=": wiki.creativecommons.org/Considerations_for_licensors|\.">: wiki.creativecommons.org/Considerations_for_licensors</alt></p>
```

Comment:
Note the the current text doesn't make much sense; the sentence only works with the link. It was probably copied from the CC HTML version and the link lost. I think it would be fine to drop the current text entirely and thus avoid alt match, that is unless it is important to match texts previously published by SPDX (I imagine it might be).

Reply:
I'm not certain, but it probably is important for matching previous texts.  I would leave the alt text in.